publish.js

```diff
@@ -11,9 +12,9 @@ const remark = require('remark');
 const u = require('unist-builder');
 const fs = require('mz/fs');
 const fecha = require('fecha');
-require('any-observable/register/rxjs-all');
-const Observable = require('any-observable');
-const streamToObservable = require('stream-to-observable');
+const {merge, throwError} = require('rxjs');
+const {catchError, filter, finalize} = require('rxjs/operators');
```

Comment:
Why I do not see `catchError` and `finalize` usage above?

Reply:
Removed them